src/element.js

```diff
@@ -271,9 +271,23 @@ SVG.Element = SVG.invent({
         for (len = well.children.length; len--;) {
           this.node.appendChild(well.firstElementChild)
         }
-
       // otherwise act as a getter
       } else {
+        // expose node modifiers
+        if (typeof svg === 'function') {
+          this.each(function () {
+            well = svg(this)
+
+            if (well instanceof SVG.Element) {
+              this.replace(well)
+            }
+
+            if (typeof well === 'boolean' && !well) {
+              this.remove()
+            }
```

Comment:
Check for `false` expicitely and at first, then you can simplify the replace-condition

```js
            // If modifier returns false, discard node
            if (well === false) {
              this.remove()
            }
            
            // If modifier returns new node, use it
            if (well != this) {
              this.replace(well)
            }
```

Reply:
You are right!

Reply:
Oh - One thing I forgot: You have to return like `return this.remove()` or make it an else-if. Otherwise both conitions can be executed.

I think I prefer the else-if in this case

Reply:
Yes, I was checking that `well` is truthy but if-else is more readable I think :+1: